config/routes.rb

```diff
@@ -6,9 +6,16 @@
   get  'welcome/featured'
   get  'welcome/popular'
   get 'events/loading'
+  # resources :bookings
+  post '/paypal_hook' => 'bookings#paypal_hook', as: :hook
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.